src/game_legacy.h

```diff
@@ -339,7 +339,7 @@ long field_1516FB;
     long boulder_reduce_health_wall;
     long boulder_reduce_health_slap;
     long boulder_reduce_health_room;
-    struct GuiMessage messages[GUI_MESSAGES_COUNT];
+    struct GuiMessage messages_[3];
```

Comment:
This one still seems wrong.

Reply:
Why? I should maintain size of a struct consistent with DK but now it is possible to set GUI_MESSAGES_COUNT to any value.

Reply:
I see what you mean.